rest-spring-security/pom.xml

```diff
@@ -0,0 +1,302 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.camel.springboot.example</groupId>
+        <artifactId>examples</artifactId>
+        <version>4.8.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>camel-example-spring-boot-rest-spring-security</artifactId>
+    <name>Camel SB Examples :: REST DSL - SPRING SECURITY - JWT</name>
+    <description>An example showing Camel REST DSL secured with Spring Security and JWT token in a Spring Boot application
+    </description>
+
+    <properties>
+        <category>Rest</category>
+        <spring.boot-version>${spring-boot-version}</spring.boot-version>
+        <lombok-mapstruct-binding.version>0.2.0</lombok-mapstruct-binding.version>
+        <docker-maven-plugin.version>0.45.1</docker-maven-plugin.version>
+        <testcontainers-keycloak.version>3.6.0</testcontainers-keycloak.version>
+        <json-unit-assertj.version>4.1.0</json-unit-assertj.version>
```

Comment:
Can we move these up a level to the root pom.xml if they are needed?   Please check camel parent and see if there's a version there that we could use